<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zernikeEditDlg</class>
 <widget class="QDialog" name="zernikeEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>624</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Zernike Values</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,1">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="useCurrent">
         <property name="text">
          <string>Import current Wavfront</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disable">
         <property name="text">
          <string>Disable ALL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearAll">
         <property name="text">
          <string>Clear All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read">
         <property name="text">
          <string>Read from File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save">
         <property name="text">
          <string>Save to File</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Zerns MaxOrder:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxOrder">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="singleStep">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="numberOfTerms">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Wavefront size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sizeSb">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>6000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="createSurface">
            <property name="text">
             <string>Create Wavefront</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>zernikeEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>zernikeEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
